[main] VMR Credential Scanner failed

[Failing build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2591769&view=logs&j=64b157b0-b759-5b33-d0c4-ab4cdfa6bffc&t=12fd9f7f-2b95-5d26-4891-8af68b06773a) (internal Microsoft link)

`CentOS9` and `Fedora41` offline legs are failed in `Guardian: Post Analysis` step with the following message:

    ##[error]1. Credential Scanner Error CSCAN-AZURE0140 - File: dotnet-sources/src/sdk/NuGet.config. Line: 71. Column 1. 
    Signature: c788ddf047ef3242498344b4f7be7dfdf78cf4411c07b5800ba18df010ed7e1c
    Tool: Credential Scanner: Rule: CSCAN-AZURE0140 (Found Azure AD Client Access Token.). 
    {Searcher}CSCAN-AZURE0140
    {Code}See NuGet.config line 71 for the code resulting in match
    {Info}Found Azure AD Client Access Token.
    {Suggest}Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan 
    
    ##[error]3. Credential Scanner Error CSCAN-GENERAL0060 - File: dotnet-sources/src/nuget-client/docs/cross-platform-debugging.md. Line: 66. Column 1. 
    Signature: 96032a7942278c80f80c612ad9964243018c21692bdc46c0e1dc4c5211d77d60
    Tool: Credential Scanner: Rule: CSCAN-GENERAL0060 (Found General Password.). 
    {Searcher}CSCAN-GENERAL0060
    {Code}See cross-platform-debugging.md line 66 for the code resulting in match
    {Info}Found General Password.
    {Suggest}Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan 
    
    ##[error]5. Credential Scanner Error CSCAN-MSFT0090 - File: dotnet-sources/src/aspire/tests/Aspire.Hosting.RabbitMQ.Tests/AddRabbitMQTests.cs. Line: 95. Column 1. 
    Signature: 4b0a8464bfee519af469cfd8e6daf4e87812eb37041be937b03bf01fb530c1d2
    Tool: Credential Scanner: Rule: CSCAN-MSFT0090 (Found Internal Common Default Password.). 
    {Searcher}CSCAN-MSFT0090
    {Code}See AddRabbitMQTests.cs line 95 for the code resulting in match
    {Info}Found Internal Common Default Password.
    {Suggest}Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan 
    
    ##[error]28. Credential Scanner Error CSCAN-GENERAL0020 - File: dotnet-sources/src/aspnetcore/src/Shared/TestCertificates/https-aspnet.key. Line: 1. Column 1. 
    Signature: 6856b12427c9850dd79a2548ff00a07f72ea6e3c1051241d74cc2858c2a56fa7
    Tool: Credential Scanner: Rule: CSCAN-GENERAL0020 (Found X.509 Certificate Private Key.). 
    {Searcher}CSCAN-GENERAL0020
    {Code}See https-aspnet.key line 1 for the code resulting in match
    {Info}Found X.509 Certificate Private Key.
    {Suggest}Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan 
    
    ##[error]29. Credential Scanner Error CSCAN-GENERAL0130 - File: dotnet-sources/src/aspire/tests/Aspire.Dashboard.Tests/Integration/OtlpGrpcServiceTests.cs. Line: 129. Column 1. 
    Signature: c15b92d6114fa2843b301acf148c8d9719433dc3b4cc02bdd4cbbd2003225318
    Tool: Credential Scanner: Rule: CSCAN-GENERAL0130 (Found Client Secret / Api Key.). 
    {Searcher}CSCAN-GENERAL0130
    {Code}See OtlpGrpcServiceTests.cs line 129 for the code resulting in match
    {Info}Found Client Secret / Api Key.
    {Suggest}Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan

--- a/eng/pipelines/templates/jobs/vmr-build.yml
+++ b/eng/pipelines/templates/jobs/vmr-build.yml
@@ -172,7 +172,7 @@ jobs:
   # We either build the repo directly, or we extract them outside (which is what partners do)
   - ${{ if parameters.buildFromArchive }}:
     - name: sourcesPath
-      value: $(Build.ArtifactStagingDirectory)/dotnet-sources/
+      value: $(Agent.TempDirectory)/dotnet-sources/
   - ${{ else }}:
     - name: sourcesPath
       value: $(vmrPath)

